<!--
  Launches the infrastructure running on a robot. 
 -->
<launch>

  <param name="/use_sim_time" value="true" />

  <arg name="robot_name" default="robot"/>
  <arg name="robot_type" default="pc"/> 
  <arg name="robot_rapp_whitelist" default="[]"/>
  <arg name="robot_concert_whitelist" default="[]"/>

  <include file="$(find rocon_app_manager)/launch/concert_client.launch">
    <arg name="robot_name" value="$(arg robot_name)"/>
    <arg name="concert_whitelist" value="$(arg robot_concert_whitelist)"/>
    <arg name="robot_type" value="$(arg robot_type)"/>
    <arg name="rapp_package_whitelist" value="$(arg robot_rapp_whitelist)" />
    <arg name="rapp_package_blacklist" value="[]" />
    <arg name="concert_watch_period" value="1"/> <!-- flip faster, convenient -->
    <arg name="firewall" value="false"/>
    <arg name="robot_unique_name" value="false"/>
    <arg name="local_machine_only" value="true"/>
    <arg name="screen" value="true" />
  </include>

</launch>
